AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/ValidatorParameterHelper.java

```diff
@@ -465,6 +469,61 @@ private String getWrapperType() {
 		}
 	}
 
+	public class AnyOfPrimitiveOrWrapperParameterRequirement extends BaseParameterRequirement {
+
+		private List<TypeKind> types;
+
+		public AnyOfPrimitiveOrWrapperParameterRequirement(TypeKind... types ) {
+			this.types = Arrays.asList(types);
+		}
+
+		@Override
+		protected String description() {
+			StringBuilder builder = new StringBuilder();
+			builder.append("Any of the primitives: [");
+			for (int i = 0; i < types.size() - 1; ++i) {
+				builder.append(getWrapperType(types.get(i))).append(", ");
+			}
+			builder.append(getWrapperType(types.get(types.size() - 1))).append(" ]");
```

Comment:
I think it would be better to move this to do `for` loop and check with an `if` for the last comma. And just add `]` outside the loop.